aur/repos2aur

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if (( $# == 0 )); then
+  echo "error: No arguments passed."
+  echo "pkgbase is a required argument"
+  exit 1
+fi
+
+PKGBASE=$1
+
+if [[ ! -d "$PKGBASE" ]]; then
+  echo "error: directory $1 does not exists"
+  exit 1
+fi
+
+TMPDIR=$(mktemp -d)
+
+function cleanup {
+  rm -rf "$TMPDIR"
+}
+
+trap cleanup EXIT
```

Comment:
$TMPDIR is already an existing variable; mktemp creates directories relative to it. Please don't use the same name. :p 

I recommend:

```
trap 'rm -rf "$working_dir"' EXIT
working_dir=$(mktemp -d)
```